frontend/src/lib/lemon-ui/icons/icons.tsx

```diff
@@ -2383,6 +2383,17 @@ export function IconDatabase(props: LemonIconProps): JSX.Element {
     )
 }
 
+export function IconPipeline(props: LemonIconProps): JSX.Element {
```

Comment:
Where's this icon from?

Reply:
where should we take icons from?

Reply:
Ask @corywatilo 😄 For products (which this arguably is in some way) we should be using the same ones as on the website